docs/UserGuide.md

```diff
@@ -677,7 +677,7 @@ This glossary provides definitions for the special terms used in this user guide
 
 **Recurring Session:** Repeating session that occurs every number of days.
 
-**URL:** The address of a web page.
+**URL:**  Uniform Resource Locator, is an address of a web page. e.g. **https://www.google.com/**
```

Comment:
Haha honestly I think putting Uniform resource locator is even more confusing to ppl. Although putting the example is a good idea ah.

Reply:
Haha wait but that's how the acronym URL came about.

Reply:
Okay amended god hao!